src/covid_model_seiir_pipeline/regression/task.py

```diff
@@ -1,68 +1,74 @@
 from argparse import ArgumentParser, Namespace
-
-import logging
 from pathlib import Path
 import shlex
 from typing import Optional
 
+from covid_shared.cli_tools.logging import configure_logging_to_terminal
+from loguru import logger
 import numpy as np
 import pandas as pd
 
 from covid_model_seiir_pipeline.math import compute_beta_hat
-from covid_model_seiir_pipeline.static_vars import REGRESSION_SPECIFICATION_FILE, INFECTION_COL_DICT
+from covid_model_seiir_pipeline.static_vars import REGRESSION_SPECIFICATION_FILE
 from covid_model_seiir_pipeline.regression.data import RegressionDataInterface
 from covid_model_seiir_pipeline.regression.specification import RegressionSpecification
 from covid_model_seiir_pipeline.regression import model
 
-log = logging.getLogger(__name__)
-
 
 def run_beta_regression(draw_id: int, regression_version: str) -> None:
     # Build helper abstractions
+    logger.info('Beginning regression.')
     regression_spec_file = Path(regression_version) / REGRESSION_SPECIFICATION_FILE
     regression_specification = RegressionSpecification.from_path(regression_spec_file)
     data_interface = RegressionDataInterface.from_specification(regression_specification)
     # Load data
+    logger.info('Loading regression data.')
     location_ids = data_interface.load_location_ids()
-    location_data = data_interface.load_all_location_data(location_ids=location_ids,
-                                                          draw_id=draw_id)
+    location_data = data_interface.load_infection_data(draw_id, location_ids)
+    location_data = {location_id: location_data[location_data['location_id'] == location_id].copy()
+                     for location_id in location_ids}
     covariates = data_interface.load_covariates(regression_specification.covariates, location_ids)
 
-    # Run ODE fit
+    logger.info('Preparing inputs.')
     np.random.seed(draw_id)
     beta_fit_inputs = model.ODEProcessInput(
         df_dict=location_data,
-        col_date=INFECTION_COL_DICT['COL_DATE'],
-        col_cases=INFECTION_COL_DICT['COL_CASES'],
-        col_pop=INFECTION_COL_DICT['COL_POP'],
-        col_loc_id=INFECTION_COL_DICT['COL_LOC_ID'],
-        col_lag_days=INFECTION_COL_DICT['COL_ID_LAG'],
-        col_observed=INFECTION_COL_DICT['COL_OBS_DEATHS'],
+        col_date='date',
+        col_infections='infections_draw',
+        col_pop='population',
+        col_loc_id='location_id',
+        col_lag_days='duration',
+        col_observed='observed_deaths',
```

Comment:
What led to moving away from `INFECTION_COL_DICT`?

Reply:
It's a half-baked abstraction.  There's a lot of inconsistent use of indirection in the regression code.  This is a struct that unpacks two other structs that itself exists simply to be unpacked again immediately in the line 51 call. 

I'm not planning on sticking with raw strings in the long run.  Just want to strip out some abstraction so I can plan more easily.